modules/ROOT/pages/whats-new-in-dw.adoc

```diff
@@ -1,64 +1,204 @@
-  ** xref:dw-strings-functions-last.adoc[last]
-  ** xref:dw-strings-functions-levenshteindistance.adoc[levenshteinDistance]
-  ** xref:dw-strings-functions-lines.adoc[lines]
-  ** xref:dw-strings-functions-mapstring.adoc[mapString]
-  ** xref:dw-strings-functions-remove.adoc[remove]
-  ** xref:dw-strings-functions-replaceall.adoc[replaceAll]
-  ** xref:dw-strings-functions-reverse.adoc[reverse]
-  ** xref:dw-strings-functions-somecharacter.adoc[someCharacter]
-  ** xref:dw-strings-functions-substring.adoc[substring]
-  ** xref:dw-strings-functions-substringby.adoc[substringBy]
-  ** xref:dw-strings-functions-substringevery.adoc[substringEvery]
-  ** xref:dw-strings-functions-words.adoc[words]
-* xref:dw-coercions.adoc[dw::util::Coercions] is a new DataWeave module with functions and types to simplify data type transformations, for example, the function `toString`.
-* xref:dw-math.adoc[dw::util::Math] is a new DataWeave module with functions and variables that provides mathematical functions, for example, the function `sin`.
-* xref:dw-periods.adoc[dw::util::Periods] is a new DataWeave module with functions and types to simplify the creation of Periods, for example, the function `days`.
-* xref:dw-tree.adoc[dw::util::Tree] introduces new functions:
-  ** xref:dw-tree-functions-filterarrayleafs.adoc[filterArrayLeafs]
-  ** xref:dw-tree-functions-filterobjectleafs.adoc[filterObjectLeafs]
-  ** xref:dw-tree-functions-filtertree.adoc[filterTree]
-  ** xref:dw-tree-functions-isarraytype.adoc[isArrayType]
-  ** xref:dw-tree-functions-isattributetype.adoc[isAttributeType]
-  ** xref:dw-tree-functions-isobjecttype.adoc[isObjectType]
-* xref:dw-values-types.adoc[dw::util::Values] provides a new type `UpdaterValueProvider`.
+DataWeave introduces the following:
+
+* xref:dw-dtd.adoc[] module
+** xref:dw-dtd-functions-doctypeasstring.adoc[] function 
+** xref:dw-dtd-types.adoc[] types
+
+The following functions are updated or new in this release:
+
+* xref:dw-binaries.adoc[] updates: 
+** xref:dw-binaries-functions-frombase64.adoc[]
+** xref:dw-binaries-functions-fromhex.adoc[]
+** xref:dw-binaries-functions-tobase64.adoc[]
+* xref:dw-binaries-functions-concatwith.adoc[] (new in 2.5)
+
+* xref:dw-coercions.adoc[] updates:
+** xref:dw-coercions-functions-toarray.adoc[]
+** xref:dw-coercions-functions-tobinary.adoc[]
+** xref:dw-coercions-functions-todate.adoc[]
+** xref:dw-coercions-functions-todatetime.adoc[]
+** xref:dw-coercions-functions-tolocaldatetime.adoc[]
+** xref:dw-coercions-functions-tolocaltime.adoc[]
+** xref:dw-coercions-functions-tonumber.adoc[]
+** xref:dw-coercions-functions-tostring.adoc[]
+** xref:dw-coercions-functions-totime.adoc[]
+
+
+* xref:dw-core.adoc[] updates:
+** xref:dw-core-annotations.adoc[]
+** xref:dw-core-functions-distinctby.adoc[]
+** xref:dw-core-functions-filter.adoc[]
+** xref:dw-core-functions-filterobject.adoc[]
+** xref:dw-core-functions-groupby.adoc[]
```

Comment:
TODO CHK: 

Signature changes from 
`groupBy<T, R&#62;&#40;items: Array<T&#62;, criteria: &#40;item: T, index: Number&#41; &#45;&#62; R&#41;: { &#40;R&#41;: Array<T&#62; } `

to this

`groupBy<T, R&#62;&#40;items: Array<T&#62;, criteria: &#40;item: T, index: Number&#41; &#45;&#62; R&#41;: {&#124; &#40;R&#41;: Array<T&#62; &#124;}`